### PR TITLE
1 project setup

### PR DESCRIPTION
**Pull request by Fikerte-T**
In this milestone,

  - added project setup for the app.
  -  created file structure.
  - set up routing using React Router.
  
-No styling is added yet

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Books = () => <h2>Books page</h2>;
+
+export default Books;
```
Comment (AshakaE) [inline src/components/Books.js]: - [ ] Please follows the ducks pattern, `Books.js` should be in a folder called `books`

File: src/components/Categories.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <h2>Under construction</h2>;
+
+export default Categories;
```
Comment (AshakaE) [inline src/components/Categories.js]: - [ ] Please follows the ducks pattern, `Catergories.js` should be in a folder called `categories`

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Fikerte-T  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js (hunk 2 above)
Comment (Fikerte-T) [reply src/components/Categories.js]: Hello @AshakaE,
I followed this pattern(the duck pattern) for the redux files. Is it necessary to put each component in separate files?
/src
    |---/components
    |---/redux
        |--- /books
                | books.js
        |--- /categories
                | categories.js
        | configureStore.js
    | index.js

**Review by hiromataba — APPROVED** [review]
## STATUS: APPROVED :tada: :tada: :green_circle: 

Hi :wave: @Fikerte-T  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**